### ISSUE TITLE
Book Your Travel

### ISSUE THREAD

**Issue by mikeal** [mikeal/nodeconf2014#32]
If you're traveling in from out of town please book soon as the cost of flights will be increasing soon enough.

If you are coming from out of the country it may be easier for me to just buy the flight outright than reimburse you through a foreign bank.

If you are in the US just buy your flight and send me the receipt and I'll send you a check w/ billpay. It should arrive in a week or so.

**Comment by hackygolucky** [comment]
So was it the 7th that we'll be hanging out on the beach or the 6th? Just
read back through the exchange and I'm still a bit confused.

**Comment by mikolalysenko** [comment]
Yeah, I would also like to get one last confirmation of the dates before I finalize my travel arrangements.

**Comment by mikeal** [comment]
https://github.com/mikeal/nodeconf2014/issues/29

**Comment by mikolalysenko** [comment]
Got it!  Thanks!

**Comment by mikeal** [comment]
Australians! I'm worried about you the most :) @rvagg @timoxley @hughsk

**Comment by hughsk** [comment]
I just moved to Amsterdam, luckily enough :) It's left me a little short on disposable income, though should be able to take care of it soon. Will let you know!

**Comment by mikeal** [comment]
@hughsk if you need me to I can buy the tickets outright if you send me all your info and the flights you want.

**Comment by timoxley** [comment]
heh, Rod is the only one actually living in Australia at the moment. I'll book my flights from Singapore today.

**Comment by timoxley** [comment]
@mikeal turns out we all might be already in the US prior to nodeconf. Will let you know when plans are set.

**Comment by thlorenz** [comment]
@mikeal I booked my ticket, where do I forward the receipt to for reimbursement?

**Comment by mikeal** [comment]
just email me directly with the receipt and your address and phone number if you have a US address. if not we can use paypal or something.
